app/config/logging-cfg-local.yml

```diff
@@ -20,6 +20,12 @@ loggers:
     level: DEBUG
     handlers:
       - console
+  botocore:
+    level: INFO
+  boto3:
+    level: INFO
+  s3transfer:
+    level: INFO
```

Comment:
What about the dev configuration ? Do we want to have debug level there ?

Reply:
> See small comments in code.
> 
> `make test` is not working anymore (done from a clean repo):
> 
> ```
> test_add_asset_with_invalid_data (tests.test_admin_page.AdminAssetTestCase) ... ERROR
> Destroying test database for alias 'default' ('test_service_stac_local')...
> Traceback (most recent call last):
>   File "/home/ltshb/repositories/service-stac/app/manage.py", line 22, in <module>
>     main()
>   File "/home/ltshb/repositories/service-stac/app/manage.py", line 18, in main
>     execute_from_command_line(sys.argv)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
>     utility.execute()
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/core/management/__init__.py", line 395, in execute
>     self.fetch_command(subcommand).run_from_argv(self.argv)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/core/management/commands/test.py", line 23, in run_from_argv
>     super().run_from_argv(argv)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/core/management/base.py", line 330, in run_from_argv
>     self.execute(*args, **cmd_options)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/core/management/base.py", line 371, in execute
>     output = self.handle(*args, **options)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/core/management/commands/test.py", line 53, in handle
>     failures = test_runner.run_tests(test_labels)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/test/runner.py", line 699, in run_tests
>     result = self.run_suite(suite)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/test/runner.py", line 641, in run_suite
>     return runner.run(suite)
>   File "/usr/lib/python3.7/unittest/runner.py", line 176, in run
>     test(result)
>   File "/usr/lib/python3.7/unittest/suite.py", line 84, in __call__
>     return self.run(*args, **kwds)
>   File "/usr/lib/python3.7/unittest/suite.py", line 122, in run
>     test(result)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/test/testcases.py", line 243, in __call__
>     self._setup_and_call(result)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/django/test/testcases.py", line 279, in _setup_and_call
>     super().__call__(result)
>   File "/usr/lib/python3.7/unittest/case.py", line 676, in __call__
>     return self.run(*args, **kwds)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/moto/core/models.py", line 102, in wrapper
>     self.stop()
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/moto/core/models.py", line 86, in stop
>     self.default_session_mock.stop()
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/mock/mock.py", line 1563, in stop
>     return self.__exit__(None, None, None)
>   File "/home/ltshb/.local/share/virtualenvs/service-stac-dhB_Mi4-/lib/python3.7/site-packages/mock/mock.py", line 1529, in __exit__
>     if self.is_local and self.temp_original is not DEFAULT:
> AttributeError: '_patch' object has no attribute 'is_local'
> ```
> 
> Also it seems that the current version of boto3 is 1.19.35 while in the lockfile it uses 1.19.32. Which means the next one that issue `pipenv install --dev` gets changes in the lockfile. I would suggest to take these changes as part of this PR.

found the root issue: https://github.com/spulec/moto/issues/3535